lib/salus/scanners/semgrep.rb

```diff
@@ -39,6 +39,19 @@ def run
           # semgrep has the following behavior:
           #   ouputs a json object with a 'results' and 'errors' key
 
+          if !match['config'].nil?
+            override_keys = %w[pattern language message]
```

Comment:
Is it `language` or `languages`? It also looks like there is both `pattern` and `patterns` allowed in the semgrep config 🤔

Reply:
* salus.yaml only supports `language`, semgrep yaml only supports `languages`. 
  - semgrep yaml requires `languages` and will override `language` from salus.yaml
* salus.yaml only supports `pattern`, semgrep yaml supports `pattern` and `patterns`
  - semgrep yaml requires either `pattern` or `patterns`, and will override `pattern` from salus.yaml